### PR TITLE
Add lookup cache keys for files

### LINKED ISSUE #6: Add new cache_file_lookup_paths argument
Update the nitro cli reference to the commit including this feature
We should also deprecate `env_var_lookup_keys` in favor of `cache_env_var_lookup_keys`

### PR DESCRIPTION
**Pull request by jpgarcia**
This PR closes #6

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -115,10 +115,22 @@ inputs:
...
       is_expand: false
+  - cache_env_var_lookup_keys:
+    opts:
+      title: "List of env var keys for lookup to determine cache fingerprint (Deprecated, use `cache_env_var_lookup_keys` instead)"
+      summary: "A list of env variable keys to lookup in order to determine whether the build should be cached or not."
+      description: "A list of env variable keys to lookup in order to determine whether the build should be cached or not."
+      is_expand: false
+  - cache_file_lookup_paths:
+    opts:
+      title: "List of files for lookup to determine cache fingerprint (Deprecated, use `cache_env_var_lookup_keys` instead)"
```
Comment (ignacioola) [inline step.yml]: Here we should remove the "deprecated" message

Comment (jpgarcia) [reply step.yml]: 😱 good catch!

**Review by ignacioola — APPROVED** [review]

**Pull request merged**